packages/account-v2/src/pages/ConnectedApps/ConnectedApps.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
-import { useFetchConnectedApps } from '@deriv/api-v2';
+import React, { useCallback, useState } from 'react';
+import { useFetchConnectedApps, useRevokeConnectedApps } from '@deriv/api-v2';
 import { Loader } from '@deriv-com/ui';
+import { ConnectedAppsRevokeModal } from './ConnectedAppsRevokeModal';
 import { ConnectedAppsTable } from './ConnectedAppsTable';
 
 export const ConnectedApps = () => {
     const { data: connectedApps, isLoading } = useFetchConnectedApps();
+    const { mutate: revokeMutate } = useRevokeConnectedApps();
```

Comment:
since useFetchConnectedApps and useRevokeConnectedApps are referring to the same thing, can't we create a single hook for fetch and mutation ?
You would probably some need to invalidate the useFetchConnectedApps call  on mutation success?

Reply:
Not really, both are diff api calls. We fetch new apps when mutation is successful. That needs to be handled in the main component. We need to check for error for both calls. Not sure if that can be done in one hook? Also, i feel since both have different functionality why not have separate hooks?